src/app/spaces/[id]/committee/_components/space-comments.tsx

```
@@ -81,19 +103,91 @@ function CommentInfo({ comment }: { comment: SpaceComment }) {
+        {showReplyInput && (
+          <div className="mt-2 flex flex-col gap-2">
+            <textarea
+              className="bg-neutral-800 border border-neutral-600 p-2 rounded-md text-white text-sm"
+              rows={2}
+              placeholder="Write your reply..."
+              value={replyText}
+              onChange={(e) => setReplyText(e.target.value)}
+            />
+            <button
+              onClick={handleReply}
+              className="self-end  bg-primary  p-3 rounded-full text-sm hover:text-[#1E1E1E] flex items-center gap-1"
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)